osu.Game/Screens/Play/HUDOverlay.cs

```diff
@@ -156,6 +158,13 @@ public HUDOverlay(DrawableRuleset drawableRuleset, IReadOnlyList<Mod> mods, bool
                     Padding = new MarginPadding(44), // enough margin to avoid the hit error display
                     Spacing = new Vector2(5)
                 },
+                SpectatorFlow = new FillFlowContainer
```

Comment:
Does this need to be a separate thing? Could you not just use `LeaderboardFlow`?